pkg/apis/api.acorn.io/v1/types.go

```diff
@@ -325,6 +326,11 @@ type Config struct {
 	AcornDNSEndpoint             *string        `json:"acornDNSEndpoint" name:"acorn-dns-endpoint" usage:"The URL to access the Acorn DNS service"`
 }
 
+type EncryptionKeySpec struct {
```

Comment:
Why does this end in "Spec"? Generally, we use the Spec suffix to denote a struct that is literally the Spec field for a particular time. That's not the case with this type. So, unless I'm misunderstanding its point, i'd drop Spec from the name.

Reply:
It was the non-k8s native object I was using when going down the path of adding Acorn APIs. I can remove it.

Reply:
Removed the spec suffix.